src/dcat-ap/dcat-formatters.ts

```diff
@@ -46,9 +46,10 @@ export function formatDcatDataset(dcatDataset: any, template: DatasetFormatTempl
 
   const formattedDataset = adlib(template, dcatDataset, transforms);
 
-  setLeftoverInterpolations(formattedDataset);
+  // reset uninterpolated, non-editable fields since customers cannot remove them
+  resetUninterpolatedPaths(formattedDataset, nonEditableFieldPaths);
```

Comment:
When I revisited `setLeftoverInterpolations` for the first time, I could not for the life of me remember why we had this function. I eventually figured it out, but if I (who approved the original PR) had a hard time remembering the purpose of the function, I guarantee you that others will as well.

Changing the name to something like `resetUninterpolatedPathsOfUneditableFields` seemed way too verbose, so I opted to create a generic function name, explicitly pass in `nonEditableFieldPaths` and add in a clarifying comment. I'm hoping this semantic change will be a little more friendly for future devs (including ourselves).

Reply:
Fine by me